packages/ckeditor5-widget/tests/widgetresize.js

```diff
@@ -139,29 +138,25 @@ describe( 'WidgetResize', () => {
 		} );
 	} );
 
-	describe( 'visibility', () => {
+	describe( 'selectability', () => {
 		beforeEach( () => {
 			createResizer();
 		} );
+		it( 'handles resizer selection on view element focus', () => {
```

Comment:
There should be two tests here.

Test 1 - whether `select` and `deselect` works correctly. So something like it is here already. I even think that you could create `Resizer` instance "by hand" instead of taking it from view, to simplify the case. Then, select/deselect it and check if its `isSelected` changes and if `.seletedResizer` changes.

Test 2 - check how changing selection works. Remove selection (use view writer, you can set selection to `null`). Check if `selectedResizer` is `null`. Select the view element with resizer. Check if `select()` was called. De-select again and check if `deselect()` was called.

Actually, test 3 would be needed as well - whether calling `select()` when something is selected properly deselects the already selected resizer.